msbuild/Xamarin.iOS.Tasks.Windows/Xamarin.iOS.Common.After.targets

```diff
@@ -403,6 +403,10 @@ Copyright (C) 2011-2013 Xamarin. All rights reserved.
 			<Output TaskParameter="LinkerOutputItems" ItemName="_XamarinLinkerItems" />
 		</Xamarin.MacDev.Tasks.ILLink>
 
+		<PropertyGroup>
+			<!-- Sometimes the ILLink task itself fails, in which case the _ILLinkExitCode value won't be set. In this case, set it to a value that will cause the expected error to be shown later on. -->
```

Comment:
Is there a reason why this is the case? Does ILLink need a fix and if so, is there an issue for it?

Reply:
There are any number of reasons the ILLink task may fail for legitimate reasons, an easy one being that the network went down.